packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -169,7 +169,7 @@ export class SummarizerNode implements IRootSummarizerNode {
         const summary = new SummaryNode({
             ...localPathsToUse,
             referenceSequenceNumber: this.wipReferenceSequenceNumber,
-            basePath: parentPath,
+            basePath: parentSkipRecursion ? this._latestSummary?.basePath ?? parentPath : parentPath,
```

Comment:
Can you add a comment here explaning why this specific code was added because its not obvious. You can add a link to the bug and the PR too.